demosplan/DemosPlanCoreBundle/ResourceTypes/OrgaResourceType.php

```diff
@@ -215,6 +220,13 @@ protected function getProperties(): array
             $this->createToManyRelationship($this->departments)->readable(false, static function (Orga $orga): TightencoCollection {
                 return $orga->getDepartments();
             }),
+            $this->createAttribute($this->isPlanningOrganisation)->readable(true,
+                function (Orga $orga): bool {
+                    return $orga->hasType(OrgaType::MUNICIPALITY, $this->globalConfig->getSubdomain())
+                        || $orga->hasType(OrgaType::PLANNING_AGENCY, $this->globalConfig->getSubdomain())
+                        || $orga->hasType(OrgaType::HEARING_AUTHORITY_AGENCY, $this->globalConfig->getSubdomain());
```

Comment:
This is fine, I just would like to point out that we have created a currentContextProvider (during the whole addon-work) which also provides the currentCustomer. It might make the code a bit better readable as it is clear that we're talking about the current customers subdomain and not any other subdomain from the global config. Just to give you the option in case you wanna try that. This is not a blocking concern.

Reply:
I will try but make another PR for this, to avoid slow down this one.
https://github.com/demos-europe/demosplan-core/pull/1141